src/Console/Base/EnvironmentCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Cmf.CustomerPortal.Sdk.Common;
+using Microsoft.Extensions.Configuration;
+using System.CommandLine;
+
+namespace Cmf.CustomerPortal.Sdk.Console.Base
+{
+    class EnvironmentCommand : BaseCommand
+    {
+        public EnvironmentCommand(string name, string description) : base(name, description)
+        {
+            Add(new Option<string>(new string[] { "--destination", "--dest" }, Resources.DESTINATIONENVIRONMENT_HELP));
+        }
```

Comment:
Actually, we are using this parameter to load the correct _appsettings.\<destination\>.json_ file

Reply:
Ok i see we are still using the destination parameter to select the settings file. This could also have been done using an environment variable. But if we are using it this, there are several commands that extending **BaseCommand** and no **EnvironmentCommand**, including the login command... this way we cannot use a different app settings file in these commands

Reply:
You're right, I've already noticed it. I'm working on it.

Reply:
Commit fc38858.
**--destination** parameter removed. Now the file _appsettings.json_ is always the one loaded.